### ISSUE TITLE
New CSSharp version broke retakes.cfg

### ISSUE THREAD

**Issue by Porcellacion** [B3none/cs2-retakes#114]
Hi,

After the new CSSharp version, v213, the retakes.cfg is being executed too early.
```
Apr 19 07:54:11:  execing cs2-retakes/retakes.cfg
Apr 19 07:54:11:  exec: couldn't exec '{*}cfg/skill1.cfg', unable to read file
Apr 19 07:54:11:  execing server_default.cfg
Apr 19 07:54:11:  execing server.cfg
Apr 19 07:54:11:  execing gamemode_casual.cfg
```

and is being overwritten by the default server configurations.

Just a heads up.

**Comment by Snailpower** [comment]
Seconded, gonna roll back to v212 for now

**Comment by Snailpower** [comment]
Another workaround seems to be to find the last cfg and just putting an exec in there like this:

exec cs2-retakes/retakes.cfg

That way it overrides for instance your execing gamemode_casual.cfg

I had the problem that cs2-retakes would be overwritten with the cfg of the gamemode, this fixed that

**Comment by B3none** [comment]
This is fixed in the `2.0.3` release
